### PR TITLE
Add CODEOWNERS

### PR DESCRIPTION
**Pull request by hparra**
See https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

For full power, the main branch needs to be protected with added rules. See https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches

Addresses: [MWPW-130633](https://jira.corp.adobe.com/browse/MWPW-130633)

Test URLs:

- https://hparra-patch-1--milo--adobecom.hlx.page/

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Milo CODEOWNERS
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+/libs/blocks/video-metadata/ @hparra
```
Comment (auniverseaway) [inline CODEOWNERS:4]:
```suggestion
/libs/utils/ @adobecom/admins
/libs/blocks/video-metadata/ @hparra
```

Comment (auniverseaway) [reply CODEOWNERS:4]: Small add for the time being. This is a great start.

**Review by auniverseaway — APPROVED** [review]
Small suggestion to get this party started.

**Review by mokimo — APPROVED** [review]
Cool find - that's a great hidden feature 👍

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Milo CODEOWNERS
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+/libs/utils/ @adobecom/admins
+/libs/blocks/video-metadata/ @hparra
+/libs/blocks/global-navigation/ @overmyheadandbody @mokimo @narcis-radu
```
Comment (seanchoi-dev) [inline CODEOWNERS:6]:
```suggestion
/libs/blocks/global-navigation/ @overmyheadandbody @mokimo @narcis-radu
/libs/blocks/faas/ @seanchoi-dev
/libs/blocks/faas-config/ @seanchoi-dev

```

File: CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+/libs/blocks/faas/ @seanchoi-dev
+/libs/blocks/faas-config/ @seanchoi-dev
+
```
Comment (rgclayton) [inline CODEOWNERS:9]:
```suggestion
/libs/blocks/carousel/ @rgclayton 
```

Comment (ryanmparrish) [inline CODEOWNERS:9]:
```suggestion
/libs/blocks/accordion/ @fullcolorcoder @ryanmparrish
/libs/blocks/aside/ @Sartxi  
/libs/blocks/how-to/ @fullcolorcoder @ryanmparrish 
/libs/blocks/icon-block/ @elan-tbx 
/libs/blocks/marquee/ @ryanmparrish @Sartxi @auniverseaway 
/libs/blocks/media/ @ryanmparrish
/libs/blocks/quote/ @ryanmparrish
/libs/blocks/tabs/ @ryanmparrish
/libs/blocks/text/ @ryanmparrish
/libs/blocks/z-pattern/ @ryanmparrish
```

Comment (seanchoi-dev) [inline CODEOWNERS:9]:
```suggestion
/libs/blocks/local-nav/ @seanchoi-dev

```

Comment (npeltier) [reply CODEOWNERS:9]: /libs/blocks/merch/ @yesil @3ch023 @honstar @VKniaz @npeltier

Comment (npeltier) [reply CODEOWNERS:9]: please add 
`/libs/blocks/merch/ @yesil @3ch023 @honstar @VKniaz @npeltier `

Comment (Brandon32) [inline CODEOWNERS:9]:
```suggestion
/libs/blocks/marketo/ @Brandon32 
/libs/blocks/table-of-contents/ @Brandon32
/libs/blocks/chart @meganthecoder @Brandon32 @JasonHowellSlavin @sanjayms01 
/libs/blocks/pdf-vewer/ @sanjayms01 
```
Thanks!

**Comment by hparra** [comment]
I'm merging this now as GH can't handle overlapping suggestions well.

**Pull request merged**